javascript/modules/client/src/client-dao.ts

```diff
@@ -12,12 +12,48 @@ import {
 import {
   DAOFactory__factory,
   Registry__factory,
+  DAOFactory,
+  TokenFactory,
 } from "@aragon/core-contracts-ethers";
+import { BigNumber } from "@ethersproject/bignumber";
+import { BigNumberish } from "ethers";
 
 export { ICreateDaoERC20Voting, ICreateDaoWhitelistVoting };
 
 export class ClientDaoERC20Voting extends ClientCore
   implements IClientDaoBase, IClientDaoERC20Voting {
+  /** Helpers */
+  params = {
+    create: (
```

Comment:
Should this be a separate private helper function? Or at least a static function within the class?